publication-doi-commons/src/test/java/no/unit/nva/publication/doi/dto/PublicationStreamRecordTestDataGeneratorTest.java

```diff
@@ -0,0 +1,79 @@
+package no.unit.nva.publication.doi.dto;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.notNullValue;
+
+import com.github.javafaker.Faker;
+import no.unit.nva.publication.doi.dto.PublicationStreamRecordTestDataGenerator.Builder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class PublicationStreamRecordTestDataGeneratorTest {
+
+    public static final String ENTITY_DESCRIPTION = "entityDescription";
+    public static final String ENTITYDESCRIPTION_REFERENCE = "reference";
+    public static final String ENTITYDESCRIPTION_REFERENCE_PUBLICATION_INSTANCE = "publicationInstance";
+    public static final String TYPE = "type";
+    public static final String MAIN_TITLE = "mainTitle";
+    public static final String CONTRIBUTORS = "contributors";
+    public static final String DATE = "date";
+    public static final String DATE_YEAR = "year";
+    public static final String DATE_MONTH = "month";
+    public static final String DATE_DAY = "day";
+    public static final String STATUS = "status";
+    public static final String IDENTIFIER = "identifier";
+    public static final String DASH = "-";
+    private static Builder validStreamRecordBuilder;
+
+    @BeforeEach
+    void setUp() {
+        validStreamRecordBuilder = Builder.createValidPublication(new Faker());
+    }
+
+    @Test
+    void asDynamoDbStreamRecord() {
+        var streamRecord = validStreamRecordBuilder.build().asDynamoDbStreamRecord();
+        assertThat(streamRecord.getEventID(), notNullValue());
+        assertThat(streamRecord.getEventSourceARN(), notNullValue());
+        assertThat(streamRecord.getEventID(), notNullValue());
+        assertThat(streamRecord.getAwsRegion(), notNullValue());
+        assertThat(streamRecord.getEventName(), notNullValue());
+        assertThat(streamRecord.getEventVersion(), notNullValue());
+        //assertThat(streamRecord.getUserIdentity(), notNullValue());
```

Comment:
Dead code?

Reply:
@janbj : It doesn't get populated , because I don't know what to populate. So more leaving a reminder.  Maybe prefix comment with a  TODO?

Reply:
I added a TODO.